frontend/src/hooks/localLabels.ts

```diff
@@ -3,7 +3,9 @@ import { useAddonData } from "./addon";
 import { AliasData, isRandomAlias } from "./api/aliases";
 
 export type LocalLabel = {
+  /** `type` is to allow for a transition period while people migrate to a new add-on version that uses `mask_type` */
```

Comment:
i.e. a version that includes this: https://github.com/mozilla/fx-private-relay-add-on/pull/344